tests/test_backend.py

```diff
@@ -355,6 +355,23 @@ def test_delete(self):
         res = self.cache.delete("a")
         self.assertFalse(bool(res))
 
+    def test_delete_return_value(self):
+        """returns a boolean instead of int since django version 3.1"""
+        self.cache._dj_ver_maj = 4
+        self.cache._dj_ver_min = 0
+        self.cache.set("a", 1)
+        res = self.cache.delete("a")
+        self.assertTrue(res is True)
```

Comment:
there is no need for `assertTrue` with `res is True`, just `assertTrue(res)` is enough

Reply:
done - added a check on the type
```
        self.assertEqual(type(res), bool)
        self.assertTrue(res)
```